templates/awesome.html

```diff
@@ -0,0 +1,60 @@
+{% extends "base.html" %}
+
+{% block content %}
+<div class="awesome">
+    {% for subsection in section.subsections %}
+    {% set section = get_section(path=subsection) %}
+
+    <h1 id="{{ section.title | slugify }}">
+        {{ section.title }}<a class="anchor-link" href="#{{ section.title | slugify }}">#</a>
+    </h1>
+    <div class="card-list padded-content">
+        {% for post in section.pages %}
+        <div class="card" id="{{ post.title | slugify }}">
+            {% if post.extra.image %}
+            <div class="card-image card-image-default">
+                <img src="{{ get_url(path=post.extra.image) }}" />
+            </div>
+            {% endif %}
+            <div class="card-text">
+                <a class="card-title" href="{{ post.extra.link }}">
```

Comment:
I think clicking any part of the card should link to the `post.extra.link`. Can we wrap the whole card in an `<a>` element?